logstash-core/lib/logstash/output_delegator.rb

```diff
@@ -62,7 +135,24 @@ def events_received
     @events_received.value
   end
 
+  # There's no concept of 'busy' workers for a threadsafe plugin!
   def busy_workers
-    @worker_queue.size
+    if @threadsafe
+      0
+    else
+      @workers.size - @worker_queue.size
+    end
+  end
+
+  private
+
+  # Needed for tests
+  def threadsafe_worker
```

Comment:
attr_reader can be declared below a private keyword and AFAIK the generated methods are private